modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -868,6 +868,23 @@ definition {
 		AppBuilderAdmin.deleteEntry(itemName = "my custom object");
 	}
 
+	@description = "Delete an existing Form View with no active publishing through the kebab/action menu"
+	@priority = "5"
+	test DeleteFormViewThroughActionMenu {
+		property portal.acceptance = "true";
```

Comment:
Hey @nkkwon , could you please change it to "false"? Thanks!

Reply:
Pushed newest changes